orchestrator/services/process_broadcast_thread.py

```diff
@@ -66,35 +72,58 @@ def stop(self) -> None:
         self.is_alive()
 
 
-def api_broadcast_process_data(request: Request) -> BroadcastFunc | None:
+def _nop(_process_id: UUID) -> None:
+    pass
+
+
+def _broadcast_ws_fn(process_id: UUID) -> None:
+    # Catch all exceptions as broadcasting failure is noncritical to workflow completion
+    try:
+        websocket_data = create_process_websocket_data(process_id)
+        logger.info("Send process data to ws", data=websocket_data)
+        send_process_data_to_websocket(process_id, websocket_data)
+    except Exception as e:
+        logger.exception(e)
+
+
+def _broadcast_queue_put_fn(broadcast_queue: queue.Queue, process_id: UUID) -> None:
+    # Catch all exceptions as broadcasting failure is noncritical to workflow completion
+    try:
+        websocket_data = create_process_websocket_data(process_id)
+        logger.info("Putting data in queue", data=websocket_data)
+        broadcast_queue.put((str(process_id), websocket_data))
```

Comment:
i think that the ProcessDataBroadcastThread should be changed to call `send_process_data_to_websocket()` as well, as right now it wouldn't send the cache invalidation event. (seems like a bug in the existing code)

But see my other comment first about whether or not to remove process data entirely..